### ISSUE TITLE
maven support

### ISSUE THREAD

**Issue by azachar** [conventional-changelog/standard-version#322]
Hello, 
Do you plan to support maven based projects too? Is there any proof of concept available?

Thank you!

Best regards,
Andrej

**Comment by nexdrew** [comment]
Could you elaborate on what you mean by "maven support"? Are you mainly thinking of bumping a version in pom.xml or build.grade(.kts)?

**Comment by azachar** [comment]
Yep Andrew, only bumping a correct version into the pom is sufficient and I do not want to reinvent the wheel.

I found https://github.com/conveyal/maven-semantic-release

But it seems to me that it does a lot of magic that I do not need, moreover it does not use this plugin ;(

Best regards,
Andrej

**Comment by jbottigliero** [comment]
If it's just a matter of bumping/altering the semantic version string in a specific file, I think we can merge this request into #313.

**Comment by bcoe** [comment]
seems reasonable to me to bump maven meta-information.
